src/modules/call/ui/components/call-connect.tsx

```
@@ -0,0 +1,91 @@
+        currentCall.leave();
+        currentCall.endCall();
+        setCall(undefined);
+      }
+    };
+  }, [client, meetingId]);
+
+  if (!client || !call) {
+    return (
+      <div className="from-sidebar-accent to-sidebar flex h-screen items-center justify-center bg-radial">
+        <LoaderIcon className="size-6 animate-spin text-white" />;
+      </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)